controllers/processVideo.go

```diff
@@ -6,6 +6,6 @@ import (
 	"github.com/gin-gonic/gin"
 )
 
-func ProcessVideo(c *gin.Context) {
+func ProcessVideo(_ *gin.Context) {
 	fmt.Println("Do Something")
 }
```

Comment:
We can remove this for now as it does nothing, you can add it later on whenever you starts working on it again

Reply:
fixed it in local, will raise the PR